#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_shebang_fixes.dpatch by Jindrich Novy <jnovy@redhat.com>
##
## DP: fixes shebang lines /usr/env to /usr/bin/env

@DPATCH@
--- ./scripts/context/ruby/wwwserver.rb.orig	2007-02-14 01:05:17.000000000 +0100
+++ ./scripts/context/ruby/wwwserver.rb	2007-02-14 01:05:28.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/env ruby
+#!/usr/bin/env ruby
 
 banner = ['WWWServer', 'version 1.0.0', '2003-2006', 'PRAGMA ADE/POD']
 
--- ./scripts/context/ruby/ctxtools.rb.orig	2007-02-14 01:05:37.000000000 +0100
+++ ./scripts/context/ruby/ctxtools.rb	2007-02-14 01:06:03.000000000 +0100
@@ -1737,7 +1737,7 @@
 
     def add_shebang(filename,program)
         unless self[0] =~ /^\#/ then
-            self.insert(0,"\#!/usr/env #{program}")
+            self.insert(0,"\#!/usr/bin/env #{program}")
         end
         unless self[2] =~ /^\#.*?copyright\=/ then
             self.insert(1,"\#")
